web/packages/shared/components/AccessRequests/NewRequest/RequestCheckout/RequestCheckout.tsx

```diff
@@ -208,131 +233,178 @@ export function RequestCheckout<T extends PendingListItem>({
+                  <Box>{item.name}</Box>
                 </Flex>
-              )}
-              {userGroupFetchAttempt?.status === 'failed' && (
-                <Danger mt={4}>{userGroupFetchAttempt.statusText}</Danger>
-              )}
-              {userGroupFetchAttempt?.status === 'success' &&
-                appsGrantedByUserGroup.length > 0 && (
-                  <AppsGrantedAccess apps={appsGrantedByUserGroup} />
-                )}
-              {isResourceRequest && (
-                <ResourceRequestRoles
-                  roles={resourceRequestRoles}
-                  selectedRoles={selectedResourceRequestRoles}
-                  setSelectedRoles={setSelectedResourceRequestRoles}
-                  fetchAttempt={fetchResourceRequestRolesAttempt}
+                <CrossIcon
+                  clearAttempt={clearAttempt}
+                  item={item}
+                  toggleResource={toggleResource}
+                  createAttempt={createAttempt}
                 />
-              )}
-              <Box mt={6} mb={1}>
-                <SelectReviewers
-                  reviewers={dryRunResponse?.reviewers.map(r => r.name) ?? []}
-                  selectedReviewers={selectedReviewers}
-                  setSelectedReviewers={setSelectedReviewers}
-                />
-              </Box>
-              <Validation>
-                {({ validator }) => (
+              </Flex>
+              <KubeNamespaceSelector
+                kubeClusterItem={item}
+                savedResourceItems={data}
+                toggleResource={toggleResource}
+                fetchKubeNamespaces={fetchKubeNamespaces}
+                bulkToggleKubeResources={bulkToggleKubeResources}
+                namespaceRequired={
+                  requiresNamespaceSelect &&
+                  affectedKubeClusterName.includes(item.id)
+                }
+              />
+            </Flex>
+          </Flex>
+        </td>
+      );
+    }
+  }
+
+  return (
+    <Validation>
+      {({ validator }) => (
+        <>
+          {showRequestRoleErrBanner &&
+            fetchResourceRequestRolesAttempt.status === 'failed' && (
+              <Alert
+                kind="danger"
+                children={fetchResourceRequestRolesAttempt.statusText}
+              />
+            )}
+          {hasUnsupportedKubeRequestModes && (
+            <Alert kind="danger">
+              You can only request Kubernetes resource kind{' '}
+              {unsupportedKubeRequestModes} for cluster{' '}
+              {affectedKubeClusterName}, but the web UI does not support these
+              kinds yet. Use the{' '}
+              <ExternalLink
+                target="_blank"
+                href="https://goteleport.com/docs/admin-guides/access-controls/access-requests/resource-requests/#step-26-search-for-resources"
+              >
+                tsh CLI tool
+              </ExternalLink>{' '}
+              to create this particular request.
```

Comment:
It'd be nice to, if possible, show an example tsh invocation to achieve that. Otherwise not only we're not allowing the user to accomplish what they wanted to accomplish, but we also force them to figure out on their own how to do this through tsh.

Reply:
how about this:

<img width="487" alt="image" src="https://github.com/user-attachments/assets/bf1fa82d-66f2-492b-b47c-61cbf0030f54">


copied command: `tsh request search --kind=pod --kube-cluster=pumpkin-kube-cluster --all-kube-namespaces`

Reply:
This is pretty nice. 👍